### PR TITLE
appsec: go setup fixes

### PR DESCRIPTION
**Pull request by Julio-Guerra**
### What does this PR do?

Fix issues in the documentation of the setup AppSec for Go where the step of manually instrumenting Go apps was missing. 

### Motivation

Avoid support issues.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by Hellzy — APPROVED** [review]
LGTM 👍

File: content/en/security/application_security/enabling/go.md [hunk 1]
```diff
@@ -25,23 +25,25 @@ further_reading:
...
+1. **Add to your program's go.mod dependencies** the latest version of the Datadog Go library (version 1.53.0 or later):
 
    ```console
    $ go get -v -u gopkg.in/DataDog/dd-trace-go.v1
    ```
-   To check that your service's language and framework versions are supported for ASM capabilities, see [Compatibility][2].
 
-2. **Recompile your program** and enable ASM:
+2. Datadog has a series of pluggable packages which provide out-of-the-box support for instrumenting a series of Go libraries and frameworks.
+   A list of these packages can be found in the [compatibility requirements][1] page. Import these packages into your application and follow the configuration instructions listed alongside each [integration][1].
```
Comment (estherk15) [inline content/en/security/application_security/enabling/go.md:39]:
```suggestion
   A list of these packages can be found in the [compatibility requirements][1] page. Import these packages into your application and follow the configuration instructions listed alongside each integration.
```

File: content/en/security/application_security/enabling/compatibility/go.md [hunk 2]
```diff
@@ -34,83 +34,63 @@ The minimum tracer version to get all supported ASM capabilities for Go is 1.50.
...
+The Go packages listed in this page are those especially interesting for ASM. You can also find more tracing intergrations in [APM's tracing compatibility page][16].
 
-- Attacker source HTTP request details
-- Tags for the HTTP request (status code, method, etc)
-- Distributed Tracing to see attack flows through your applications
+**Note**: The [Go integrations documentation][6] provides a detailed overview of the supported packages and their APIs, along with usage examples.
 
-##### ASM Capability Notes
-- **Vulnerability Management for Code-level** is not supported
+<div class="alert alert-info">If you don't see your library of choice listed, let us know! Fill out <a href="https://forms.gle/gHrxGQMEnAobukfn7">this short form to send details</a>.</div>
```
Comment (estherk15) [inline content/en/security/application_security/enabling/compatibility/go.md:53]:
```suggestion
<div class="alert alert-info">If you don't see your library of choice listed, fill out <a href="https://forms.gle/gHrxGQMEnAobukfn7">this form to send details</a>.</div>
```

File: content/en/security/application_security/enabling/compatibility/go.md [hunk 3]
```diff
@@ -34,83 +34,63 @@ The minimum tracer version to get all supported ASM capabilities for Go is 1.50.
...
 You must be running Datadog Agent v5.21.1+
 
 Starting from tracer version 1.53.0, ASM does not require [CGO][15].
 
 ## Integrations
 
+The Go tracer includes support for the following frameworks, data stores and libraries.
 
-### Web framework compatibility
+The Go packages listed in this page are those especially interesting for ASM. You can also find more tracing intergrations in [APM's tracing compatibility page][16].
```
Comment (estherk15) [inline content/en/security/application_security/enabling/compatibility/go.md:49]:
```suggestion
The Go packages listed in this page are relevant for ASM. You can also find more tracing integrations in [APM's tracing compatibility page][16].
```

**Review by estherk15 — APPROVED** [review]
Left some suggestions, thanks!

**Pull request merged**